charts/assemble-backstage/templates/postgres-pvc.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.postgres.persistent }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name:  {{ include "assemble-backstage.fullname" . }}-postgres
+  labels:
+    paas.redhat.com/appcode: "ASBL-001"
```

Comment:
It's probably not harmful that this is here, but it would be nice if we could define custom labels so this doesn't show up for all users.

Reply:
Fixed